src/Bicep.Core/Analyzers/Linter/Rules/EnvironmentUrlHardcodedRule.cs

```diff
@@ -35,47 +35,57 @@ public override void Configure(IConfigurationRoot config)
         }
 
         protected override string FormatMessage(params object[] values)
-            => string.Format("{0} -- Found: [{1}]", this.Description, values.First());
+            => string.Format("{0} -- Found: '{1}'", this.Description, values.First());
```

Comment:
>-- [](http://example.com/codeflow?start=34&length=2)

The `--` structure is unusual for Bicep messages. I'd replace with something like this: `"{0} The disallowed host that was found: \"{0}\""`

Reply:
=> string.Format("{0} Found this disallowed host: '{1}'", this.Description, values.First());